### DISCUSSION TITLE
Terma & Syarat (Tech Writing)

### DISCUSSION THREAD

**Discussion by rohaizanr** [Dev4w4n/e-masjid.my#195] [General]
Hasil karangan terma dan syarat akan disertakan di bahagian copyright.

User yang guna sistem ini, maksudnya dah terima terma & syarat.

Akta PDPA,

dan lain-lain untuk protect user dan e-masjid.my

**Comment by rohaizanr** [comment]
@mfar1984 expected delivery 2 weeks ok ?

**Comment by mfar1984** [comment]
> @mfar1984 expected delivery 2 weeks ok ?

Sorry baru reply. Semalam biz trip sampai jumaat. اِ نْ شَآ ءَ اللّهُ boleh

**Comment by rohaizanr** [comment]
@mfar1984 terima kasih

**Comment by mfar1984** [comment]
[DASAR PRIVASI.docx](https://github.com/Dev4w4n/e-masjid.my/files/14833144/DASAR.PRIVASI.docx)
[Terma dan Syarat PDPA.docx](https://github.com/Dev4w4n/e-masjid.my/files/14833145/Terma.dan.Syarat.PDPA.docx)
[TERMA PERKHIDMATAN.docx](https://github.com/Dev4w4n/e-masjid.my/files/14833146/TERMA.PERKHIDMATAN.docx)

dalam terma dan syarat tu. 

kalau masa mendaftar, minta gunakan terma dan syarat PDPA

**Comment by rohaizanr** [comment]
Great work @mfar1984 !

**Comment by mfar1984** [comment]
> Great work @mfar1984 !

Nanti sy amend mengenai terma dan syarat tu, diharap benda ni boleh di edit atau ditambah di bahagian dashboard > tetapan > Terma & Syarat

dibawah tu akan ada data privasi, data keselamatan, polisi, Dasar Penggunaan, Pendaftaran dan lain2 depend dgn web yang sedia ada nanti
